src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -613,6 +613,50 @@ const logFilterDeprecation: ConfigDeprecation = (
   }
 };
 
+const logFormatDeprecation: ConfigDeprecation = (
+  settings,
+  fromPath,
+  addDeprecation,
+  { branch }
+) => {
+  addDeprecation({
+    configPath: 'logging',
```

Comment:
`configPath` isn't really relevant here since this is a general warning that is only indirectly related to configuration, but since it is required I set it to `logging`.

The alternative would be to register this deprecation elsewhere in core (i.e. not as a `ConfigDeprecation`), but placing it here alongside the other logging deprecations requires fewer changes and keeps things in one place.